apiary.apib

```diff
@@ -3549,6 +3549,29 @@ Add a draft work order.
             + type: `workOrder` (string)
             + id: `7c1d8672-f502-4333-9ea4-7a45add15115` (string)
 
+### workOrders.update [POST /workOrders.update]
+
+Update a draft work order.
```

Comment:
PP: `Update a drafted work order.`

Reply:
I think it's fine like this though. "Draft invoice" or "Draft work order" is a valid way to describe it.